Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/BackgroundWorkScheduler.kt

```diff
@@ -17,7 +18,11 @@ import java.util.concurrent.ExecutionException
  * @see BackgroundConstants
  * @see BackgroundWorkHelper
  */
-object BackgroundWorkScheduler {
+object BackgroundWorkScheduler : BackgroundWorkSchedulerBase() {
+
+    fun init(component: ApplicationComponent) {
```

Comment:
Where is this called?

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/2505/files#diff-fdd468b18ccf30dc7a16638bc7f0032deedf65a479ca506ee7406a6380f9cc6fR78 in onCreate of CoronaWarnApplication